src/test/test-ask-password-api.c

```diff
@@ -10,10 +10,13 @@ static void ask_password(void) {
         _cleanup_strv_free_ char **ret = NULL;
 
         r = ask_password_tty(-1, "hello?", "da key", 0, 0, NULL, &ret);
-        assert(r >= 0);
-        assert(strv_length(ret) == 1);
-
-        log_info("Got %s", *ret);
+        if (r == -ECANCELED)
+                assert_se(ret == NULL);
```

Comment:
Why `assert_se()` instead of `assert()` here?

Reply:
Because we're in test here.

Reply:
Ouch. Sorry for the noise.